src/Transport/Curl.php

```diff
@@ -467,7 +467,7 @@ private function setup_handle($url, $headers, $data, $options) {
 	 * @param string $response Response data from the body
 	 * @param array $options Request options
 	 * @return string|false HTTP response data including headers. False if non-blocking.
-	 * @throws \WpOrg\Requests\Exception
+	 * @throws \WpOrg\Requests\Exception If the request resulted in a Curl error.
```

Comment:
This probably shouldn't be the PSR-4 class notation, but the actual name of the project:
```suggestion
	 * @throws \WpOrg\Requests\Exception If the request resulted in a cURL error.
```